MSBuild race to write portable assembly attributes file

Many of our builds are failing with the following:

``` txt
error MSB3491:  Could not write lines to file "d:\j\workspace\roslyn_master---e026a511\Binaries\Temp\.NETPortable,Version=v5.0.AssemblyAttributes.cs". 
The process cannot access the file 'd:\j\workspace\roslyn_master---e026a511\Binaries\Temp\.NETPortable,Version=v5.0.AssemblyAttributes.cs' because it is being used by another process. 
[d:\j\workspace\roslyn_master---e026a511\src\Tools\Source\CompilerGeneratorTools\Source\CSharpErrorFactsGenerator\CSharpErrorFactsGenerator.csproj] ->      
6>C:\Program Files (x86)\MSBuild\14.0\bin\Microsoft.Common.CurrentVersion.targets(2992,5):
```

This is a race condition in MSBuild writing out the temporary files which was previously won't fixed.  This is causing build breaks as we are promoting this to an error.  For now leaving this as a warning that we ignore.  

Root cause: http://vstfdevdiv:8080/DevDiv2/DevDiv/_workItems?id=403105&_a=edit

--- a/build/Targets/Imports.targets
+++ b/build/Targets/Imports.targets
@@ -341,6 +341,7 @@
        ==================================================================================== -->
 
   <PropertyGroup>
+    <TargetFrameworkMonikerAssemblyAttributesPath>$(IntermediateOutputPath)$(TargetFrameworkMoniker).AssemblyAttributes$(DefaultLanguageSourceExtension)</TargetFrameworkMonikerAssemblyAttributesPath>
     <PostCompileBinaryModificationSentinelFile>$(IntermediateOutputPath)$(TargetFileName).pcbm</PostCompileBinaryModificationSentinelFile>
     <OptimizationDataFolderPath>$(NuGetPackageRoot)\RoslynDependencies.OptimizationData\2.0.0-rc-61101-16\content\OptimizationData</OptimizationDataFolderPath>
     <OptimizationDataFile>$([System.IO.Path]::GetFullPath('$(OptimizationDataFolderPath)\$(TargetName).pgo'))</OptimizationDataFile>

